openshift/app.bc.yaml

```diff
@@ -61,11 +61,12 @@ objects:
           uri: "${SOURCE_REPO_URL}"
         type: Git
         dockerfile: |-
-          FROM mhart/alpine-node:10
+          FROM mhart/alpine-node:12
```

Comment:
does this mean we all need to add node 12 and set our development environments to 12 for this project?

Reply:
This is mainly more to ensure our production environments are using the latest Node LTS version - as of right now we are not using any features introduced in 12, and I don't suspect we would be either, so 10 and 12 should be swappable on dev environment.